test/internal/utils.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 OST.com Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
```

Comment:
Any specific reason for not keeping test case of Internal contract constructor?
I know it will be very small test case but 1 advantage of having constructor test cases is in case constructor parameter gets changed, we have a unit test to validate.
Let me know your thoughts.

Reply:
Internal contract's constructor doesn't set any variable. It passes a parameter to Organized contract. There will be duplication for constructor's unit test cases of Organized and Internal contract. 
@jasonklein what do you think ?